File: Patterns/go/pattern4.go
// n = 4

// 1111
// 2222
// 3333
// 4444

package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)
	
	row := 1
	for row <= n {
		col := 1
		for col <= n {
			fmt.Printf("%d", row)
			col++
		}
		row++
		fmt.Printf("\n")
	}
}

File: Patterns/python/pattern4.py
# n = 4
# 4321
# 4321
# 4321
# 4321

n = int(input())
row = 1
while row <= n:
    col = 1
    val = n
    while col <= n:
        print(val, end="")
        col += 1
        val -= 1
    row += 1
    print("")